### PR TITLE
Test that appcache is not supported in shared workers

### PR DESCRIPTION
**Pull request by domenic**
Follows https://github.com/whatwg/html/pull/2384.

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/204742867)

# Firefox (nightly channel) #
Testing web-platform-tests at revision f243ec7c86d8e1df788308d682bf57877b06a5ad
Using browser at version BuildID 20170222110216; SourceStamp 9a9db410f20781076424307265decbc5de1e94cc
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5001/html/browsers/offline/no-appcache-in-shared-workers-historical.html">/html/browsers/offline/no-appcache-in-shared-workers-historical.html</a></summary>


|                Subtest                 | Results |
|----------------------------------------|---------|
|                                        | OK      |
| `self.applicationCache must not exist` | PASS    |
| `ApplicationCache must not be exposed` | PASS    |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/204742869)

# Chrome (unstable channel) #
Testing web-platform-tests at revision f243ec7c86d8e1df788308d682bf57877b06a5ad
Using browser at version 58.0.3018.3 dev
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5001/html/browsers/offline/no-appcache-in-shared-workers-historical.html">/html/browsers/offline/no-appcache-in-shared-workers-historical.html</a></summary>


|                Subtest                 | Results |
|----------------------------------------|---------|
|                                        | OK      |
| `self.applicationCache must not exist` | PASS    |
| `ApplicationCache must not be exposed` | PASS    |

</details>
</details>

File: html/browsers/offline/no-appcache-in-shared-workers-historical.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<script>
+"use strict";
+
+if (typeof SharedWorker === "undefined") {
+  test(() => {
+    assert_equals(typeof SharedWorker, "undefined");
```
Comment (foolip) [inline html/browsers/offline/no-appcache-in-shared-workers-historical.html]: Given the outer `if`, I think no assertion at all here would be clearer.

**Review by foolip — APPROVED** [review]

**Review by PaulinaParangerHr — COMMENTED** [review]
Nice

**Review by mathieubathalon — APPROVED** [review]

**Review by Bonniemarie216 — APPROVED** [review]

**Comment by gagancheema867-ops** [comment]
+92 323 0029670

**Pull request merged**